README.md

```diff
@@ -2,6 +2,21 @@
 Superset is fast, lightweight, intuitive, and loaded with options that make it easy for users of all skill sets to explore and visualize their data.
 
 ## Relations
+### Redis
+Redis acts as both a cache and message broker for Superset services. It's a requirement to have a redis relation in order to start the Superset application.
+```
+# deploy redis charm
+juju deploy redis-k8s --edge
+
+# relate redis charm
+juju relate redis-k8s superset-k8s
+
+# remove relation
```

Comment:
Perhaps this is only necessary for ```CONTRIBUTING.md```, the instructions here should be for deployment only.

Reply:
Take your point, I've updated this for `README.md`